.gitignore

```diff
@@ -1,7 +1,8 @@
+.idea/
```

Comment:
pycharm's configuration file, will be automatically git with, so have to ignore it.